packages/plugins/plugin-sketch/src/components/SketchContainer.tsx

```diff
@@ -13,6 +14,18 @@ import { Sketch, type SketchProps } from './Sketch';
 const SketchContainer = ({ classNames, sketch, ...props }: SketchProps) => {
   const id = fullyQualifiedId(sketch);
   const hasAttention = useHasAttention(id);
+  const dispatch = useIntentDispatcher();
+
+  const onThreadCreate = useCallback(() => {
+    void dispatch({
+      // TODO(Zan): We shouldn't hardcode the action ID.
```

Comment:
yeah, likely the way this should work is that Sketch's toolbar is extensible and Thread can add this tool to the toolbar